package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "nf start",
-    "dev:oidc": "VUE_APP_OIDC_ENABLED=true npm run dev",
-    "build": "vue-cli-service build",
+    "dev": "nf start vite,nginx",
+    "dev:build": "nf start build,nginx",
```

Comment:
what are the use cases of this?

Reply:
This is meant to work similarly to how it does today. Files rebuild automatically, but there's no HMR. Personally, sometimes I like to have a browser tab open that reflects one version of the code and another tab open that reflects another. I can compare the two. In that case, I like that the browser doesn't try to automatically refresh either tab as I make changes.